appinventor/appengine/src/com/google/appinventor/client/TopToolbar.java

```diff
@@ -312,6 +319,9 @@ public void updateFileMenuButtons(int view) {
         buildDropDown.setItemEnabled(MESSAGES.showExportAndroidAab2(), false);
       }
     } else { // We have to be in the Designer/Blocks view
+      if (fileDropDown.getName() == "ProjectDesignOnly") {
```

Comment:
Technically we should use `.equals` here for correct Java semantics since `==` compares for identity. We just get lucky that GWT does the wrong thing when converting to JavaScript.